web/app/auth/page.tsx

```diff
@@ -1,28 +1,29 @@
 'use client';
-import { Button, Flex, Heading, Input, InputGroup, InputLeftAddon, Text } from '@chakra-ui/react';
-
+import { Button, Flex, Heading, Input, InputGroup, InputLeftAddon, Text,Icon } from '@chakra-ui/react';
+import { MdOutlineMessage } from 'react-icons/md'
 const Home = () => (
-  <Flex height={'100vh'} alignItems={'center'} justifyContent={'center'}>
-    <Flex direction={'column'}>
-      <Text fontSize="2xl" mb={10}>
-        Enter Mobile Number
-      </Text>
-      <Text fontSize="1xl" color={'gray'} mb={5}>
-        We will sent you{' '}
-        <Text as="b" color={'black'}>
-          one time password
-        </Text>{' '}
-        on this mobile number
-      </Text>
-      <InputGroup mb={5}>
-        <InputLeftAddon children="+91-" />
-        <Input variant="flushed" />
-      </InputGroup>
-      <Button size="md" height="48px" width="-moz-fit-content" border="2px" colorScheme="blue">
-        GET OTP
-      </Button>
-    </Flex>
-  </Flex>
+    <><Flex height={'25vh'} alignItems={'center'} justifyContent={'center'} mt={10}><Icon as={MdOutlineMessage} w={20} h={20}  />
+    </Flex><Flex height={'75vh'} alignItems={'Flex-start'} justifyContent={'center'}>
+            <Flex direction={'column'} >
```

Comment:
U dont need to provide {""} When u are just providing a string as attributes that makes code ugly

Remove all {} and just provide strings to attributes of chakra ui components